packages/ra-no-code/src/ResourceConfiguration/useResourceConfigurations.ts

```diff
@@ -0,0 +1,7 @@
+import { useContext } from 'react';
+import { ResourceConfigurationContext } from './ResourceConfigurationContext';
+
+export const useResourceConfigurations = () => {
```

Comment:
Why not name it `useResourceConfigurationContext`?

Reply:
That's something I don't like the react-admin hooks in general. The `Context` suffix feels weird and I've only seen it in our code base. In general, the fact that a hook relies on context is not relevant to the users